src/components/Book.js

```diff
@@ -0,0 +1,16 @@
+import './Book.css';
+import React from 'react';
+
+function Book() {
+  return (
+    <div className="book">
+      <h2 className="book-title">Book Title</h2>
+      <p className="book-author">Author</p>
+      <button className="remove-button" type="button">
+        Remove Book
+      </button>
+    </div>
+  );
+}
```

Comment:
- [ ] This component should take props from the parent component and render them inside this component.  please kindly remove the "Book Title" and "Author" text and replace them by using props